puppet/modules/kubernetes/manifests/controller_manager.pp

```diff
@@ -28,6 +29,14 @@
     $version_before_1_6 = true
   }
 
+  if $::kubernetes::controller_feature_gates == [] {
+    $_feature_gates = delete_undef_values([
+      $::kubernetes::_enable_pod_priority ? { true => 'PodPriority=true', default => undef },
```

Comment:
I don't think this is needed. Only needed for `API server, scheduler and kubelet` https://v1-10.docs.kubernetes.io/docs/concepts/configuration/pod-priority-preemption/